package.json

```diff
@@ -37,11 +38,18 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
     "babel-register": "^6.26.0",
-    "eslint": "^5.10.0",
+    "chai": "^4.2.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-15": "^1.3.0",
+    "eslint": "^5.14.1",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-react": "^7.11.1",
+    "mocha": "^6.0.2",
     "node-sass": "^4.10.0",
-    "react": "^16.6.3",
+    "react": "^15.6.2",
+    "react-dom": "^15.6.2",
```

Comment:
I recommend both react and react-dom being `peerDependencies` `>= 15` (given you actually support from v15) and then you can have the latest versions (or v15) in `devDependencies` here